pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/SegmentUploadIntegrationTest.java

```diff
@@ -81,10 +89,15 @@ protected List<String> getBloomFilterColumns() {
     return null;
   }
 
+  @BeforeMethod
+  public void setUpTest()
```

Comment:
Calling `TestUtils.ensureDirectoriesExistAndEmpty(_tempDir, _segmentDir, _tarDir)` in `setUp` is insufficient as there is another test case `testUploadAndQuery` that leaves data in the tmp directories.

Using two table names will also lead to more code for now, since it seems like `BaseClusterIntegrationTest` is written in a way that assumes one table/config per subclass, e.g. `getTableName`, `createOfflineTableConfig`. 

Hence, I prefer to keep the test structure as is with the before and after method annotations. Another way is to create a different test class altogether, but combining the two here to not prevent slow down from too many tests.